tests/e2e/pageobjects/ide/GetStarted.ts

```diff
@@ -0,0 +1,76 @@
+/*********************************************************************
```

Comment:
Wouldn't better place for this pageobject be `/tests/e2e/pageobjects/dashboard/GetStarted.ts`?

Reply:
Definitely yes, thank you =)